openshift/distro/ubi/9/Dockerfile

```diff
@@ -28,80 +28,53 @@
 
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 
-ENV HOME=/home
-WORKDIR "${HOME}/app"
-
 ARG RUBY_VERSION="3.3"
-
-RUN microdnf module enable -y "ruby:${RUBY_VERSION}" \
- && chown -R 1001:1001 "${HOME}"
-
 ARG BUILD_DEPS="tar make file findutils git patch gcc automake autoconf libtool redhat-rpm-config openssl-devel ruby-devel"
 
-RUN microdnf install -y --nodocs ${BUILD_DEPS}
-
-# Bundler should be kept as-is for productisation
-ARG BUNDLE_VERSION_MATCH=true
-# If the above is false
-ARG BUNDLE_VERSION="2.3.5"
-
-ARG BUNDLE_GEMFILE=Gemfile
-ARG BUNDLE_WITHOUT=development:test
-
-COPY --chown=1001:1001 ${BUNDLE_GEMFILE}.lock "${HOME}/app/"
-
-RUN mkdir -p "${HOME}/.gem/bin" \
- && echo "gem: --bindir ~/.gem/bin --install-dir /usr/share/gems" > "${HOME}/.gemrc"
-
-RUN gem uninstall --executables bundler \
- && BUNDLED_WITH=$(cat ${BUNDLE_GEMFILE}.lock | \
-      grep -A 1 "^BUNDLED WITH$" | tail -n 1 | sed -e 's/\s//g') \
- && if test "${BUNDLE_VERSION_MATCH}x" = "truex"; then \
-      gem install -N bundler --version "${BUNDLED_WITH}"; \
-    else \
-      gem install -N bundler --version "${BUNDLE_VERSION}"; \
-    fi \
- && echo Using $(bundle --version), originally bundled with ${BUNDLED_WITH} \
- && bundle config --local silence_root_warning 1 \
- && bundle config --local disable_shared_gems 1 \
- && bundle config --local without ${BUNDLE_WITHOUT} \
- && bundle config --local gemfile ${BUNDLE_GEMFILE}
+RUN microdnf module enable -y "ruby:${RUBY_VERSION}" \
+ && microdnf install -y --nodocs ${BUILD_DEPS}
```

Comment:
Maybe add:
```
microdnf clean all
```

So all downloaded packages aren't included in the FS layer

Reply:
updated, good point